composer.json

```diff
@@ -94,7 +94,8 @@
             "@test-reset-database"
         ],
         "test-main": "./phpunit --testsuite main",
-        "test-reset-database": "./phpunit --testsuite reset-database --bootstrap tests/bootstrap-reset-database.php"
+        "test-reset-database": "./phpunit --testsuite reset-database --bootstrap tests/bootstrap-reset-database.php",
+        "post-install-cmd": ["@composer bin phpstan install", "@composer bin phpbench install"]
```

Comment:
this way, when we'll do "composer install", both phpstan and phpbench will be installed, and no more problem with phpstan locally because it would not know about phpbench